internal/pkg/skuba/cni/cilium.go

```diff
@@ -226,6 +226,7 @@ func CreateOrUpdateCiliumConfigMap(client clientset.Interface, ciliumVersion str
 			"enable-ipv6":              "false",
 			"identity-allocation-mode": "crd",
 			"preallocate-bpf-maps":     "false",
+			"sidecar-istio-proxy-image": "istio-proxy",
```

Comment:
Does it work for you? I thought that has to be the full image name, including the repository if it's different than `docker.io`. From my understanding, this is going to be evaluated as `docker.io/istio-proxy`.

I think we should rather use something like `registry.suse.de/caasp/.../istio-proxy`, or `registry.suse.com`, depending if we run released skuba or not.

For generating the image names, you can use `GetGenericImage` function, it's used for example here:

https://github.com/SUSE/skuba/blob/98f67addb5bb7c2df56d511681b5115f5b3a9884/internal/pkg/skuba/addons/cilium.go#L41

Reply:
Yes, I had the same doubt but it works. I checked the function and it looks for `any match of the regular expression`
https://golang.org/pkg/regexp/#Regexp.MatchString

I created a dummy program to verify it:
```
package main

import (
	"fmt"
	"regexp"
)

func main() {
	matching := "istio-proxy"
	myString := "registry.suse.de/devel/caasp/4.5/containers/containers/caasp/v4.5/istio-proxyv2:1.5.9"
	myRegexp := regexp.MustCompile(matching)
	if !myRegexp.MatchString(myString) {
	    fmt.Println("No matching")
	} else {
	    fmt.Println("Matching")
	}
}
```
And returns `Matching`:
```
kosmas@kosmas:~> go run /tmp/throw.go
Matching
```

Reply:
OK, I didn't know that `sidecar-istio-proxy-image` is just a regexp. I though we require the full URI.

Reply:
But there is another issue with that line of code. You have wrong code formatting here. Run `make lint` locally to see it. ;) You need put all `=` characters on the same level.

Look for some ´gofmt` plugin in your text editor or IDE.

Reply:
Check the newest version, that should be fixed (and an error in the unit tests)